RA Scripts/Legend of Zelda, The The Minish Cap.rascript

```diff
@@ -1205,4 +1207,3 @@ rich_presence_display("{0}{1}{2}{3}{4}{5} [❤️{6}] [🗽{7}] [🧿{8}]",
     rich_presence_value("Figurines", byte(0x00AAF0)),
     rich_presence_value("Fusions", byte(0x00AB57))
 )
-
```

Comment:
I recommend readding this line to maintain adherence to POSIX (and to reduce the total number of diffs in this PR).